templates/22-loggly-nginx.conf

```diff
@@ -0,0 +1,80 @@
+# {{ ansible_managed }}
+
+# Nginx access file:
+{% for filename in loggly.nginx.access_logs %}
+  # Setup disk assisted queues. The directives are only valid for the next configured action!
+  $ActionQueueFileName fwdRule1     # unique name prefix for spool files
+  $ActionQueueSaveOnShutdown on     # save messages to disk on shutdown
+  $ActionQueueType LinkedList       # run asynchronously
+  $ActionResumeRetryCount -1        # infinite retries if host is down
+
+  $ActionSendStreamDriver gtls
+  $DefaultNetstreamDriverCAFile {{ loggly.certificate_dest }}{{ loggly.certificate_file }}
+  $ActionSendStreamDriverMode 1
+  $ActionSendStreamDriverAuthMode x509/name
+  $ActionSendStreamDriverPermittedPeer *.loggly.com
+
+  $InputFileName {{ filename }}
+  $InputFileTag nginx-access{{ loop.index }}:
+  $InputFileStateFile stat-nginx-access{{ loop.index }}
+  $InputFileSeverity info
+  $InputFilePersistStateInterval 20000
+  $InputRunFileMonitor
+
+  if $programname == 'nginx-access{{ loop.index }}' then @@logs-01.loggly.com:6514;LogglyFormatNginx
+  if $programname == 'nginx-access{{ loop.index }}' then ~
+
+{% endfor %}
+
+
+#Nginx Error files:
+{% for logfile in loggly.nginx.error_logs %}
+  $ActionQueueFileName fwdRule1     # unique name prefix for spool files
+  $ActionQueueSaveOnShutdown on     # save messages to disk on shutdown
+  $ActionQueueType LinkedList       # run asynchronously
+  $ActionResumeRetryCount -1        # infinite retries if host is down
+
+  $ActionSendStreamDriver gtls
```

Comment:
I hope loggly gives you some recognition for letting them know about the error in their docs. Load driver, then specify cert file.